### PR TITLE
OSLO integration work - update all lang term names

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
Updating **all** language terms in CPD views so they follow the desired naming convention:

- `{cpdView}\{termName}` if term name only exists on a specific view
- `{shared}\{termName}` if term is used across multiple views
-  `{shared}\{sharedComponent}\{termName}` if term is used exclusively in a shared component that is used in multiple views

**Note:** we have to do double `\\` in code to escape the backslash character or the code will break.

Doing this to follow to the recommended naming convention outlined in OSLO documentation [here](https://docs.dev.d2l/index.php/Oslo#Follow_the_OSLO_langterm_naming_convention).

This naming convention will help us sort and search OSLO terms when updating them in the language management tool.

### CODE REVIEW

**Comment by IhabAlkabary** [comment]
Overall note: The old lang term files, including the translations, are ordered in ascending order. Do we need to maintain this sort order with OSLO?

**Comment by imcdonald-d2l** [comment]
> Overall note: The old lang term files, including the translations, are ordered in ascending order. Do we need to maintain this sort order with OSLO?

OSLO documentation does not say anything about the order of lang terms so I expect this not to be an issue. Activity-editor uses OSLO and their lang terms are not sorted as well. See `en.js` file [here](https://github.com/BrightspaceHypermediaComponents/activities/blob/master/components/d2l-activity-editor/lang/en.js).

**Comment by imcdonald-d2l** [comment]
A better approach to reviewing this PR is probably to override the FRA on a test site or locally and make sure no lang terms are messed up and the FRA functions as expected. Reviewing in detail is basically impossible given it's size.

**Comment by lweber-d2l** [comment]
Just curious why each of the langterm files has 13 fewer lines than before.  Were some unused langterms identified and removed?

**Comment by imcdonald-d2l** [comment]
> Just curious why each of the langterm files has 13 fewer lines than before. Were some unused langterms identified and removed?

Yes, that's exactly it. I removed unused lang terms. That work is in commits `lang term cleanup part 1` and `remove unused lang terms part 2`

File: locales/ar-SA.json [hunk 1]
```diff
@@ -1,148 +1,136 @@
...
+   "shared\\subjectProgress\\ariaTableDescription" : "جدول يظهر التقدم المُحرَز في كل موضوع.",
+   "manageTargets\\ariaTargetsTableDescription" : "جدول بأهداف التطوير المهني المستمر الخاصة بك.",
+   "manageTargets\\backToJob" : "الرجوع إلى قائمة المسميات الوظيفية",
+   "manageTargets\\backToMyCPD" : "الرجوع إلى صفحة التطوير المهني المستمر الخاص بي",
+   "shared\\backToTeamText" : "العودة إلى فريقي",
+   "readOnlyRecord\\backToUserRecords" : "العودة إلى سجلات {userDisplayName}",
+   "shared\\cancelText" : "إلغاء",
+   "shared\\ariaChooseChoice" : "اختيار {choice}",
+   "shared\\subjectProgress\\combinedText" : "مجمّعة",
+   "shared\\subjectProgress\\completedText" : "Completed:",
```
Comment (IhabAlkabary) [inline locales/ar-SA.json:22]: Looks like an existing bug with the term not translated here.

Comment (IhabAlkabary) [reply locales/ar-SA.json:22]: Maybe we should contact the Translation team about this

Comment (imcdonald-d2l) [reply locales/ar-SA.json:22]: Yeah, hopefully it is caught in next round of serge translations. If not we can reach out to translations team

Comment (lweber-d2l) [reply locales/ar-SA.json:22]: Assuming this missing term is not a blocker for merging, I approve this base on inspection of the changes.  I didn't try the FRA override as part of review but will check each page of the tool as part of QA.

**Review by lweber-d2l — APPROVED** [review]

**Pull request merged**